src/dependency-manager/src/spec/common/interface-spec.ts

```diff
@@ -6,6 +6,7 @@ export interface InterfaceSpec {
   protocol?: string;
   username?: string;
   password?: string;
+  sticky?: string;
```

Comment:
Add the traefik labels to the docker-compose if sticky is set and add a test case to confirm it gets set.